bdk-ffi/src/bdk.udl

```diff
@@ -737,6 +737,15 @@ interface Wallet {
 
   [Throws=SqliteError]
   boolean persist(Connection connection);
+
+  /// Apply relevant unconfirmed transactions to the wallet.
+  /// Transactions that are not relevant are filtered out.
+  void apply_unconfirmed_txs(sequence<UnconfirmedTx> unconfirmed_txs);
+};
+
+dictionary UnconfirmedTx {
+    Transaction tx;
+    u64 last_seen;
 };
```

Comment:
Applying our idea of using macros when possible on changes and new things, this block here can be replaced with

```txt
typedef dictionary UnconfirmedTx;
```

and the Rust code can simply add
```rust
#[derive(uniffi::Record)]
pub struct UnconfirmedTx {
    pub tx: Arc<Transaction>,
    pub last_seen: u64,
}
```

To leverage the procedural macros.